docs/bugs_and_glitches.md

```diff
@@ -1749,6 +1750,45 @@ When a Pokémon is withdrawn from the Day-Care, its Exp. Points are reset to the
  	ret
 ```
 
+### Time Capsule glitch
+
+Generation I Pokémon that evolve through trading (Graveler, Kadabra, Machoke, and Haunter) can sidestep the check that prevents Pokémon with Generation II moves from being sent back to Generation I.
+
+To do this, they must be traded at the exact level they would learn a move in Generation II after evolving (for example, a Level 34 Graveler sent to Generation II will evolve to Golem and then immediately learn Rollout). After sending the Pokémon to Generation II, they will evolve and learn the move. From there, they can be immediately sent back to Generation I, and that will not trigger the move check. Any Generation II-exclusive moves will become glitch moves in Generation I.
```

Comment:
Pare down or remove this description, and give it a more descriptive name than "Time Capsule glitch". Maybe "Trade evolutions in the Time Capsule can send glitch moves to Gen 1".